Sources/TensorFlow/Layers/Recurrent.swift

```diff
@@ -420,12 +425,150 @@ public struct RecurrentLayer<Cell: RecurrentLayerCell>: Layer {
   }
 }
 
+public protocol Mergeable: Differentiable, AdditiveArithmetic {
+  /// - Note: Trying to use `+` instead of `sum` results in compile error,
+  ///   when Tensor is extended for Mergeable comforting.
+  ///   Stack dump is stored at https://pastebin.com/PxZTir6F.
```

Comment:
Minor rewording, with a link to the issue:
```suggestion
  /// - Note: renaming `sum` to `+` results in a compiler crash when conforming `Tensor` to
  /// `Mergeable` (SR-13229).
```